/RoboMinerControllerRos2ParamProvider:
  ros__parameters:
   # supported executor types:
      # SINGLE_THREADED(0)
      # STATIC_SINGLE_THREADED(1)
      # MULTI_THREADED(2)
    ros2_executor_type: 2
    
    # default value of 0 will use number of cpu threads
    # used only for MULTI_THREADED executor
    ros2_executor_threads_num: 4
    
    # use_local_controller_mode: True
    use_fog_of_war_status: True
    user: vladimirr11
    repository: https://github.com/vladimirr11/robotics-v1
    commit_sha: not_set
    